xray/paths.py

```diff
@@ -0,0 +1,24 @@
+class PathGenerator:
+
+    def __init__(self, wg_ifc_name: str, wg: str, test_type: str, count: int):
+        self.wg_ifc_name = wg_ifc_name
+        self.wg = wg
+        self.test_type = test_type
+        self.count = count
+
+    def base_path(self) -> str:
+        return f"results/{self.wg_ifc_name}_{self.wg.lower()}_{self.test_type}_{self.count}"
```

Comment:
Let's make this `f"results/xray_{self.wg.lower()}_{self.test_type.lower()}_{self.count}"`. 

I started naming the pcaps after the interface but the pcaps collect on all interfaces, and regardless of that, the output of the analysis should not be named after the iterface. I like having a more generic naming here, which your changes go a long way towards fixing, but I think this is one last bit that needs to be changed. And along with that, the `csv(self)` function should return `f"{self.base_path()}.csv`

A minor nit for this function too: `base_path` should be private and the way that is done in python would be to name it `_base_path`

Reply:
Good idea, fixed this